prawns/identities/schema.graphql

```diff
@@ -16,7 +16,7 @@ type SubstrateIdentity @entity {
   "address"
   account: String! @index
   "hex address"
```

Comment:
Another change I was working on was removing the `account` and make it part of the `ID` but this is ok - same as before is fine